test/issues/0062.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var assert = require('assert');
+var yaml = require('../../');
+
+
+test('Reading of properties on implicit block mapping keys (map tag on the same line with key)', function () {
+  assert.doesNotThrow(function () { yaml.load('block mapping: !!map\n  !!str implicit key: some value'); });
+});
+
+test('Reading of properties on implicit block mapping keys (map tag on a new line)', function () {
+  assert.doesNotThrow(function () { yaml.load('block mapping:\n  !!map\n  !!str implicit key: some value'); });
+});
+
+test('Reading top-level mapping with explicit type tag for key', function () {
+  assert.doesNotThrow(function () { yaml.load('!!int 16: 32'); });
+});
+
+test('Reading block mapping with explicit type tag for key', function () {
+  assert.doesNotThrow(function () { yaml.load('block mapping:\n  !!int 16: 32'); });
+});
+
```

Comment:
To be honest this test case isn't very useful in my opinion. We shouldn't just check for successful parsing, but also for correct results of the process. I mean we should have there custom types and checks for parsing results, i.e. what tags were applied to what nodes.

For example, in current state this test case will not reveal the bug I mentioned before.